src/ducks/apps/components/Partnership.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react'
+import { translate } from 'cozy-ui/react/I18n'
+import { getTranslatedManifestProperty } from 'lib/helpers'
+import ReactMarkdownWrapper from 'ducks/components/ReactMarkdownWrapper'
+
+export class Partnership extends Component {
+  handleBrokenImage = e => {
+    const imgNode = e.nativeEvent.target
+    if (imgNode) imgNode.style = 'display: none;'
```

Comment:
It does not seem very reacty to do this. IMHO it would be less surprising to use a this.setState({ errored: true }) and then checking on this in the render.

Reply:
Hum, after thinking I agree, it's better to not rely on the event here 👍